packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js

```diff
@@ -0,0 +1,169 @@
+import React, {useCallback, useState} from 'react';
+import {convert} from 'css-color-function';
+import {get, getOr} from 'lodash/fp';
+import PropTypes from 'prop-types';
+import Icon from '../../atom/icon';
+import ButtonLink from '../../atom/button-link';
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+const LearnerSkillCard = (props, context) => {
+  const {
+    'aria-label': ariaLabel,
+    skillCourses,
+    skillQuestions,
+    skillTitle,
+    skillAriaLabel,
+    completedCourses,
+    buttonReviewLabel,
+    buttonReviewAriaLabel,
+    buttonExploreLabel,
+    buttonExploreAriaLabel,
+    onReviewClick,
+    onExploreClick
+  } = props;
+  const {skin, translate} = context;
+  const [hovered, setHovered] = useState(false);
+  const primarySkinColor = getOr('#0061FF', 'common.primary', skin);
+
+  const courses = translate('courses');
+  const questions = translate('questions');
+  const skillFocus = translate('skill_focus');
+  const coursesCompleted = translate('courses_completed');
+
+  const handleMouseOver = useCallback(() => setHovered(true), [setHovered]);
+
+  const handleMouseLeave = useCallback(() => setHovered(false), [setHovered]);
+
+  const buttonReviewProps = {
+    customStyle: {
+      backgroundColor: '#FFF',
+      transition: 'background-color 0.15s ease-in-out, color 0.15s ease-in-out'
+    },
+    onReviewClick,
+    'aria-label': buttonReviewAriaLabel ? `${skillTitle}, ${buttonReviewAriaLabel}` : skillTitle,
+    label: buttonReviewLabel,
+    'data-name': 'learner-skill-card-review-button'
+  };
+
+  const buttonExploreProps = {
+    customStyle: {
+      backgroundColor: hovered ? primarySkinColor : convert(`color(${primarySkinColor} a(0.07))`),
+      color: hovered ? '#FFFFFF' : primarySkinColor,
+      transition: 'background-color 0.15s ease-in-out, color 0.15s ease-in-out'
+    },
+    onExploreClick,
+    'aria-label': buttonExploreAriaLabel ? `${skillTitle}, ${buttonExploreAriaLabel}` : skillTitle,
+    label: buttonExploreLabel,
+    'data-name': 'learner-skill-card-explore-button',
+    icon: {
+      position: 'left',
+      type: 'compass',
```

Comment:
We might have a little subject here, 

The new designs are supposed to be using fontawesome icons (that were set up in #2837).
But indeed [ButtonLink](https://github.com/CoorpAcademy/components/blob/master/packages/%40coorpacademy-components/src/atom/button-link/index.tsx) is only supporting the old Nova system.

Do you think @HuyIT9898 it's heavy to adapt ButtonLink so it can also support the font awesome icons?
(maybe by considering `icon: {name: ''}` is for fontawesome, while existing `type` while go for Nova  '../../util/button-icons')

If it's too heavy we'll check if compass nova if acceptable